covid19_sfbayarea/data/sonoma.py

```diff
@@ -117,24 +117,48 @@ def transform_cases(cases_tag: element.Tag) -> Dict[str, TimeSeries]:
 
     return { 'cases': cases, 'deaths': deaths }
 
-def transform_transmission(transmission_tag: element.Tag) -> Dict[str, int]:
+def transform_transmission(transmission_tag: element.Tag) -> Dict[str, float]:
     """
     Takes in a BeautifulSoup tag for the transmissions table and breaks it into
     a dictionary of type:
-    {'community': -1, 'from_contact': -1, 'travel': -1, 'unknown': -1}
+    {
+        "household": -1.0,
+        "workplace": -1.0,
+        "congregate_care": -1.0,
+        "gathering_small": -1.0,
+        "health_care": -1.0,
+        "gathering_large": -1.0,
+        "travel": -1.0,
+        "other": -1.0,
+        "unknown": -1.0
+    }
     """
     transmissions = {}
     rows = parse_table(transmission_tag)
+
     # turns the transmission categories on the page into the ones we're using
-    transmission_type_conversion = {'Community': 'community', 'Close Contact': 'from_contact', 'Travel': 'travel', 'Under Investigation': 'unknown'}
+    transmission_type_conversion = {
+        'Congregate Care': 'congregate_care',
+        'Health Care': 'health_care',
+        'Household': 'household',
+        'Large Gathering': 'gathering_small',
+        'Other': 'other',
+        'Small Gathering': 'gathering_large',
+        'Travel': 'travel',
+        'Unknown': 'unknown',
+        'Workplace': 'workplace'
+    }
```

Comment:
This works, but in places where I’ve written or updated similar code, I’ve lower-cased the left-hand side of this mapping to make it *slightly* more robust (so if, for example, “Congregate Care” changes to “Congregate care,” we aren’t affected).

For example in Santa Clara: https://github.com/sfbrigade/data-covid19-sfbayarea/blob/ba6ae50b9f33dbb0afda42461ac9e76650d3227f/covid19_sfbayarea/data/santa_clara.py#L175-L186

Reply:
That's smart - thanks for the suggestion. I've followed this approach in the latest changes.